src/components/Filter/index.js

```diff
@@ -61,40 +61,37 @@ export default class Filter extends Component {
   };
 
   render() {
-    const { filterRenderer, data, dataKey } = this.props;
+    const { filterRenderer, data, dataKey, filterIconRenderer } = this.props;
     return (
-      <span>
-        {filterRenderer ? (
+      <div
```

Comment:
We should ideally not wrap this inside the `div`. The filter icon should just be a `span` which is inline and dropdown is placed relevant to the entire header cell.